### ISSUE TITLE
Cosmopolitanism

### ISSUE THREAD

**Issue by darobin** [darobin/politi.es#8]
Has anyone written anything good in support?

**Comment by astearns** [comment]
This seems like the only countervailing force against tribalism. People only extend their tribe if they interact with others.

**Comment by frivoal** [comment]
@astearns If you enlarge the “tribe” from a cultural/ethnic/social group to the whole citizenry of a country, you've made some progress away from the divisive identity politics that seem (from outside) to dominate American politics. But that still leaves room for nationalism and xenophobia.

**Comment by darobin** [comment]
@astearns I agree, the problem is that I don't know of a good, pragmatic defence of it beyond "it's just nicer!" I'm convinced there is a reason-based argument to be made, and it likely has been made, it just needs finding and/or mulling over.

**Comment by astearns** [comment]
It's not always nicer. Sometimes it's challenging and/or annoying. Engaging with other people can mean compromise, re-evaluations of self, and discomfort just as much as delight and discovery. Humans need to engage to learn and grow, and that sometimes can be painful.

Just as you'd worry about a child that kept to themselves and did not interact with others, you have to worry about insular groups. Cosmopolitanism is one way to get the painful, wonderful global socialization everyone needs to be fully human.

**Comment by astearns** [comment]
Hmm, after a bit of thought I realize 'fully human' isn't the best phrasing. I mean something more like 'their best self.'
